app/controllers/locations_controller.rb

```diff
@@ -20,12 +20,12 @@ def create
 
   def show
     @location = Location.find(params[:id])
-    @date = params[:date] ? Date.parse(params[:date]) : Date.today
+    @date = params[:date] ? Date.parse(params[:date]) : Date.current
   end
 
   def open_now
-    @now = Time.now
-    @open = Timetable.where(date: Date.today)
+    @now = Time.current
+    @open = Timetable.where(date: Date.current)
```

Comment:
So it looks like this would speed it up a little bit, not v noticeable but I imagine it might be a bit more once there's more data we're pulling (before: 0.003574, after:  0.002874). Not a necessary change just a funzies thing
```ruby
@open = Timetable.where(date: Date.current)
                     .where.not(open: nil, close: nil)
                     .includes(:location)
                     .order('locations.name')
                     .select { |t| t.open_at?(@now) }
```